<?xml version="1.0"?>

<launch>
  <arg name="node_name" default="ctpoint_track"/>
  <node name="$(arg node_name)" pkg="pg_ctpoint" type="track.py" output="screen">
    <rosparam command="load" file="$(find pg_ctpoint)/launch/sort.yaml"/>
  </node>
</launch>
